framework/src/org/checkerframework/framework/flow/CFAbstractAnalysis.java

```diff
@@ -127,7 +134,22 @@ public CFValue defaultCreateAbstractValue(
             // 'no information'.
             return null;
         }
-        return new CFValue(analysis, type);
+        if (type.getKind() == TypeKind.WILDCARD) {
+            AnnotatedWildcardType wildcard = (AnnotatedWildcardType) type;
+            Set<AnnotationMirror> annos = wildcard.getExtendsBound().getAnnotations();
+            return new CFValue(analysis, annos, type.getUnderlyingType());
+        }
+        return new CFValue(analysis, type.getAnnotations(), type.getUnderlyingType());
```

Comment:
Using `getEffectiveAnnotations` should avoid having to use a special case for wildcards.